python/social_bot/tasks.py

```diff
@@ -982,12 +982,12 @@ def run(self):
                 np.array(obj_pos) - np.array(finger_pos))
             obj_lifted = obj_height / self._obj_init_height - 1.0
             gripping_feature = 0.25 * l_contact + 0.25 * r_contact + min(
-                obj_lifted, 1.0)  # encourge to lift the object by obj_height
-            gripping = (gripping_feature >= 0.999999)
+                obj_lifted, 0.5)  # encourge to lift the object by obj_height
```

Comment:
gripping_feature will not be more than 1.0 in total. half for contacts, half for lifting.